ProjectSettings/EditorBuildSettings.asset

```diff
@@ -4,14 +4,5 @@
 EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
-  m_Scenes:
-  - enabled: 1
-    path: Assets/Scenes/StartScreen.unity
-    guid: 85d359e1133588742ab242de01e601ff
-  - enabled: 1
-    path: Assets/Scenes/AdoptionCenter.unity
-    guid: c699efce584cfea4fb2bdc74800066a2
-  - enabled: 1
-    path: Assets/Scenes/LivingRoom.unity
-    guid: 1974aa80b246e0a41ba1fa3b2696791a
+  m_Scenes: []
```

Comment:
This seems to break the game for me. We should leave the scenes in the build settings